src/components/app/Home.js

```diff
@@ -251,7 +251,7 @@ class Home extends PureComponent {
             <InstallButton name="Gecko Profiler" xpiURL={LEGACY_ADDON_URL}>
               Gecko Profiler Add-on
             </InstallButton>. Then use the button added to the browser, or use
-            the following shortcuts to record a profile. The button's icon is
+            the following shortcuts to record a profile. The button’s icon is
```

Comment:
Part of the react plugin's recommended rules is to forbid single quotes, because either this is a stupid mistake with an attribute outside of its tag, or this should be a curly quote like here.